patches/0054-Fix-empty-retry_strategy-of-Batch-JobDefinitions-cau.patch

```diff
@@ -1,32 +0,0 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
This patch was redundant?

Reply:
My upstream PR finally got merged in https://github.com/hashicorp/terraform-provider-aws/pull/37500